src/app/profile/yourGiving/historicalView/historicalGift/historicalGift.tpl.html

```diff
@@ -17,7 +17,7 @@ <h4 class="mb0">{{$ctrl.gift['historical-donation-line']['designation-name']}}</
                 <li>
                   <h4>
                     {{$ctrl.gift['historical-donation-line']['amount'] | currency:'$'}}
-                    <span ng-if="$ctrl.gift['recurringdonationelement']"> &mdash; {{$ctrl.gift['recurringdonationelement']['rate']['recurrence']['interval']}}</span>
+                    <span ng-if="$ctrl.gift['recurringdonations']['donations'].length"> &mdash; {{$ctrl.gift['recurringdonations']['donations'][0]['rate']['recurrence']['interval']}}</span>
```

Comment:
Here's a wireframe I had seen showing the different frequencies:
![screen shot 2016-12-08 at 8 25 24 am](https://cloud.githubusercontent.com/assets/2905714/21011592/029e2ed4-bd20-11e6-8e77-6c572da7952a.png)